### PR TITLE
Bug/assimp

### PR DESCRIPTION
**Pull request by NaftaliD**

### CODE REVIEW

File: src/ImportExport/AssimpImportExport.cpp [hunk 1]
```diff
@@ -363,9 +369,9 @@ namespace vivid {
             node->mName = aiString(aNodeName);
             node->mNumMeshes = aMeshIndexEnd - aMeshIndexStart;
             node->mMeshes = new unsigned int[node->mNumMeshes];
-            size_t index = 0;
-            for (size_t MeshIndex = aMeshIndexStart; MeshIndex != aMeshIndexEnd; MeshIndex++) {
-                node->mMeshes[index] = MeshIndex;
+            for (size_t index = aMeshIndexStart; index != aMeshIndexEnd; index++) {
+                node->mMeshes[index] = aMeshIndexStart;
```
Comment (Beilinson) [inline src/ImportExport/AssimpImportExport.cpp:373]: what? Why not:
mMeshes[index] = index?

File: lib/ImportExport/assimp/code/Common/Exporter.cpp [hunk 2]
```diff
@@ -370,7 +370,10 @@ aiReturn Exporter::Export( const aiScene* pScene, const char* pFormatId, const c
     // format. They will likely not be aware that there is a flag in the scene to indicate
     // this, however. To avoid surprises and bug reports, we check for duplicates in
     // meshes upfront.
-    const bool is_verbose_format = !(pScene->mFlags & AI_SCENE_FLAGS_NON_VERBOSE_FORMAT) || MakeVerboseFormatProcess::IsVerboseFormat(pScene);
+    //changed from:
+    //const bool is_verbose_format = !(pScene->mFlags & AI_SCENE_FLAGS_NON_VERBOSE_FORMAT) || MakeVerboseFormatProcess::IsVerboseFormat(pScene);
+    //Meaning that when flag is defined it is taken as true, but when flag isn't defined it is checked if scene is really Verbose.
+    const bool is_verbose_format = !((pScene->mFlags & AI_SCENE_FLAGS_NON_VERBOSE_FORMAT) || !MakeVerboseFormatProcess::IsVerboseFormat(pScene));
```
Comment (Beilinson) [inline lib/ImportExport/assimp/code/Common/Exporter.cpp:376]: so what exactly is the behavior now?

File: src/ImportExport/AssimpImportExport.cpp (hunk 1 above)
Comment (NaftaliD) [reply src/ImportExport/AssimpImportExport.cpp:373]: my bad, index needs to be zero

File: lib/ImportExport/assimp/code/Common/Exporter.cpp (hunk 2 above)
Comment (NaftaliD) [reply lib/ImportExport/assimp/code/Common/Exporter.cpp:376]: it checks if the flag is set, if it is, it takes that as true, if it isn't the code double checks.

**Pull request merged**